rfc/0029-community-amendment-1.md

```diff
@@ -0,0 +1,304 @@
+  direction, excluding abstaining members from that count
+* The special closing conditions specified as part of the specific process that this vote is part of have been met, if
+  any
+
+### Interpreting the Vote
+
+When a poll has closed, the votes should be inspected. If the final vote has a margin of three votes or fewer, 
+non-voting members of the administrative board must be invited to vote on the poll, with their votes counting for 
+three. They are not required to vote if they don't feel they need to, however.
+
+If any voting members of the Community Team don't vote on the poll (unintentionally or otherwise), their vote **must be
+counted as an abstention.**
+
+THe Community Team should then tally the final vote counts, and interpret them as follows.
+
+#### Failing conditions
+
+The vote has failed when one of the following is true:
+
+* The total number of _abstentions_ **is greater than or equal to** 25% of the count of voting members of the community 
+  team, rounding up to the nearest whole number
+* The total number of _negative_ votes **is greater than or equal to** the number of _positive_ votes
+* The special failing conditions specified as part of the specific process that this vote is part of have been met, if
+  any
+
+#### Passing conditions
+
+The vote has passed when one of the following is true, but only if no failing condition has been met:
+
+* The total number of _positive_ votes **is greater than** the number of _negative_ votes
+* The special passing conditions specified as part of the specific process that this vote is part of have been met, if
+  any
+
+#### Other situations
+
+As detailed above, there should be no situations where a vote ends in a tie. If a situation arises that the voting
+process is unable to cover, the vote will fail and will need to be re-tried when the process has been amended.
+
+## Process: Electing Team Members
+
+As our primary communication space is the official Discord server, Community Team member appointments should be
+processed on Discord. Quilt's Community Team members are elected using a democratic internal process:
+
+1. Members of Quilt community spaces may apply to join the community by contacting the Community Team directly. The
+   preferred contact method for this is by making use of the ModMail system on Discord.
+   
+   * Alternatively, candidates may be put forward by other members of the Community Team, after some internal 
+     discussion. This may be a result of observations made by the Community Team, suggestions from community members,
+     or otherwise - but the Community Team should always discuss before putting forward a candidate, and they should
+     ensure that the candidate is actually interested in the position they're being put forward for.
+1. Candidates must be interviewed by a Community Manager before the election process can continue. All voting members
```

Comment:
Is making it a requirement that a CM *begins* the interview necessary? I don't see how it would be a problem since that's what's been happening, but I'm not sure strictly enforcing it is necessary either.

Reply:
I don't think it's necessarily a hard requirement, but it's something that should be a Manager responsibility rather than a Moderator one - if anyone's gonna accidentally put their foot in their mouth, it should probably be a manager haha